kahuna/public/js/components/gr-image-metadata/gr-image-metadata.js

```diff
@@ -216,6 +216,12 @@ module.controller('grImageMetadataCtrl', [
     ctrl.removeKeywordFromImages = removeXFromImages('keywords', ctrl.keywordAccessor);
     ctrl.addKeywordToImages = addXToImages('keywords', ctrl.keywordAccessor);
 
+    ctrl.subjectsAccessor = (image) => imageAccessor.readMetadata(image).subjects;
+
+    ctrl.selectedImagesHasAny = (accessor) => ctrl.selectedImages.find(
```

Comment:
I like shorter, if only coz it starts just like `selectedImages`, so kinship is readily visible. But you know that for me it could be called `susanHasStrongAppleStrudelAffinity`, so gonna ping @twrichards.

Reply:
thanks for the review, the name of this one wasn't chosen for brevity (I prefer longer/descriptive variable names) it was more for the readability at the call site...
![image](https://user-images.githubusercontent.com/19289579/207286591-1c224771-2532-46d1-bba5-94089aa3c781.png)

Reply:
makes sense, thanks!